service/controller/release/resource/argoapps/resource.go

```diff
@@ -0,0 +1,255 @@
+package argoapps
```

Comment:
Can we split this up into at least two files? Maybe the state reconciliation and `konfigureVariables` stuff in separate files.

Reply:
Sure, it'll make it more readable.